arangod/RestServer/arangod.cpp

```diff
@@ -122,7 +126,14 @@ static int runServer(int argc, char** argv, ArangoGlobalContext& context) {
         },
         [](auto& server, TypeTag<ShutdownFeature>) {
           return std::make_unique<ShutdownFeature>(
-              server, std::array{ArangodServer::id<ScriptFeature>()});
+              server,
+#ifdef USE_V8
+              std::array { ArangodServer::id<ScriptFeature>() }
+#else
+              // just a random feature specified here, so the array isn't empty.
```

Comment:
Is it affect order? Maybe shouldn't be random feature?

Reply:
Changed now so that the order should be preserved.